roles/core/pxe_stack/templates/pxe_parameters.yml.j2

```diff
@@ -1,4 +1,6 @@
 pxe_parameters:
+  http_html_path:
+  http_cgi_path:
```

Comment:
Unused variables? Might be better to introduce these in a different PR, with the related code?